src/main/java/me/lucien/minesweeper/web/GameController.java

```diff
@@ -100,4 +111,16 @@ private void checkLegitimacy(int id, String key) throws HttpException {
             throw new HttpException(HttpStatus.FORBIDDEN, "The key is wrong.");
         }
     }
+
+    public void reTimer(int id) {
```

Comment:
resetTimer? reTimer isn't good because re- is just a prefix while timer isn't a verb.